### ISSUE TITLE
Editor reloads on edit - normalization removes spaces

### ISSUE THREAD

**Issue by irahopkinson** [paranext/paranext-core#1071]
**Describe the bug**
When editing the text, sometimes the cursor is moved to the start of the verse.

**To Reproduce**

1. In the Platform.Bible menu, click **Open Scripture Editor** and select a project.
2. Place the cursor at the end of a paragraph.
3. Type a space, wait 1 second and then type a letter.
4. Now hit **Backspace**. The space is removed and the cursor jumps to the start of the verse.

**Expected behavior**
The space should be retained and the cursor should not move.

**Likely scenario**
The editor reloads because the data has changed. I believe the data has changed because the PTData normalizer in C# removes the space. When the editor reloads the cursor is set to the start of the verse.

**Comment by tjcouch-sil** [comment]
I also found this a while ago when filing all those editor bugs I filed. I guess I forgot to file this one or just neglected to do so. I had this in my notes:

> Normalization is making us lose the cursor. Type a new word, then delete the word (leave the space). The double space is reduced to one space, and the cursor disappears

**Comment by irahopkinson** [comment]
Fixed in #1245.

**Comment by tjcouch-sil** [comment]
I just tested with the version before and the version when #1245 was implemented:

- (Before fix) [Fix settings validation bug introduced with JSON-RPC change (#1259) · paranext/paranext-core@4035e6e](https://github.com/paranext/paranext-core/actions/runs/11636342599)
- (Fix) [Limit the editor webview to one USJ save to a PDP at a time · paranext/paranext-core@45e9129](https://github.com/paranext/paranext-core/actions/runs/11669532979)

I saw no difference in functionality. These reproduction steps did not cause normalization in either case. I expect something we did at some point probably caused these steps not to cause a problem with normalization, but it does seem that normalization is still happening in some contexts, hence #1245.
